ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -164,23 +164,14 @@ tests:
         OPENSHIFT_VERSION=4.14
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-4-11-periodic
-  cron: 00 02 */1 * 0,3
-  steps:
-    cluster_profile: packet-assisted
-    env:
-      ASSISTED_CONFIG: |
-        OPENSHIFT_VERSION=4.11
-      TEST_SUITE: full
-    workflow: assisted-baremetal
 - as: e2e-metal-assisted-4-12-periodic
   cron: 00 02 */1 * 0,3
   steps:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
         OPENSHIFT_VERSION=4.12
-      TEST_SUITE: full
+      TEST_SUITE: conformance
```

Comment:
What is the meaning of this ?

Reply:
it will run in `dry run` mode 
https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/baremetal/conf/assisted-baremetal-conf-commands.sh#L36